<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>853</width>
    <height>767</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QDialog#Dialog{
background-color:qlineargradient(spread:pad, x1:1, y1:1, x2:1, y2:0, stop:0 rgba(118, 118, 118, 255), stop:1 rgba(255, 255, 255, 255));}</string>
  </property>
  <widget class="QTextBrowser" name="statusBox">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>610</y>
     <width>811</width>
     <height>141</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:white</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>580</y>
     <width>151</width>
     <height>22</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 75 12pt &quot;MS Shell Dlg 2&quot;;color:white</string>
   </property>
   <property name="text">
    <string>Application Status</string>
   </property>
  </widget>
  <widget class="QScrollArea" name="scrollArea">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>130</y>
     <width>811</width>
     <height>441</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:white
</string>
   </property>
   <property name="widgetResizable">
    <bool>true</bool>
   </property>
   <widget class="QWidget" name="scrollAreaWidgetContents_2">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>805</width>
      <height>435</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>90</y>
     <width>141</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 75 18pt &quot;MS Shell Dlg 2&quot;;</string>
   </property>
   <property name="text">
    <string>Server Files</string>
   </property>
  </widget>
  <widget class="QPushButton" name="stopServerButton">
   <property name="geometry">
    <rect>
     <x>730</x>
     <y>50</y>
     <width>101</width>
     <height>34</height>
    </rect>
   </property>
   <property name="text">
    <string>Stop Server</string>
   </property>
  </widget>
  <widget class="QPushButton" name="startServerButton">
   <property name="geometry">
    <rect>
     <x>730</x>
     <y>10</y>
     <width>101</width>
     <height>34</height>
    </rect>
   </property>
   <property name="text">
    <string>Start Server</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>IP Address:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>550</x>
     <y>11</y>
     <width>41</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Port:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="ipField">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>10</y>
     <width>411</width>
     <height>34</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="portField">
   <property name="geometry">
    <rect>
     <x>600</x>
     <y>10</y>
     <width>113</width>
     <height>34</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="clientCount">
   <property name="geometry">
    <rect>
     <x>790</x>
     <y>90</y>
     <width>41</width>
     <height>34</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>646</x>
     <y>90</y>
     <width>131</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Client Connected:</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
